docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<blockquote>Let's walk through by running example of a noisy electrocardiogram(ECG).  </blockquote><p> </p><p>Maybe remove this line. </p><p> </p><blockquote>We proposed to compute the Contrast Profile only when we belive that the two following assumptions are likely to be true: </blockquote><p> </p><p>Before we start talking about the contrast profile, I think it would be a nice idea to show the data and talk about the problem. What is the problem? and why should we care about that problem?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<blockquote>Before we start talking about the contrast profile, I think it would be a nice idea to show the data and talk about the problem. What is the problem? What are we trying to find here?</blockquote><p>Add simple example in the beginning</p>